XRTK-Core/Packages/com.xrtk.core/Definitions/Controllers/Simulation/Hands/SimulatedHandControllerDataProviderProfile.cs

```diff
@@ -8,7 +8,7 @@
 namespace XRTK.Definitions.Controllers.Simulation.Hands
 {
     [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Input System/Controller Data Providers/Simulated Hand Controller Data Provider Profile", fileName = "SimulatedHandControllerDataProviderProfile", order = (int)CreateProfileMenuItemIndices.Input)]
-    public class SimulatedHandControllerDataProviderProfile : BaseMixedRealityControllerDataProviderProfile
+    public class SimulatedHandControllerDataProviderProfile : SimulatedControllerDataProviderProfile
```

Comment:
Any reason for a specific rather than generic base?  Concerned this separates this from the standard implementation